docs/tutorials/getting_started.md

```diff
@@ -0,0 +1,135 @@
+# Getting started
+
+## Prerequisites
+
+1. Python installed on your system.
```

Comment:
```suggestion
1. Python 3.11 or newer installed on your system.
```

Or some clarification saying "please check the supported versions (with a link to the page showing the supported version).

Reply:
I'd go for the first. If we bump the support it's very likely that the tutorial wouldn't work anymore and it has to get updated anyway.